leapp/workflows/__init__.py

```diff
@@ -178,6 +184,11 @@ def run(self, context=None, until_phase=None, until_actor=None, skip_phases_unti
 
         self._errors = get_errors(context)
 
+        for phase in skip_phases_until, needle_phase:
+            if not is_valid_phase(phase):
+                self.log.error('Phase {phase} does not exist in the workflow'.format(phase=phase))
+                return
```

Comment:
well you should then ensure that we exit with non 0 status code.

Reply:
AFAIR yes

Reply:
Okay then, replaced this with a CommandError.